#!/usr/bin/env python
import roslib.packages

PACKAGE = "bas_arm_driver_tester"
NODE = "bas_arm_driver_tester_node"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("joint0_value", int_t, 0, "joint0 angle joint value", 80, 0, 255)
gen.add("joint1_value", int_t, 0, "joint1 angle joint value", 80, 0, 255)
gen.add("joint2_value", int_t, 0, "joint2 angle joint value", 80, 0, 255)
gen.add("joint3_value", int_t, 0, "joint3 angle joint value", 80, 0, 255)
gen.add("joint4_value", int_t, 0, "joint4 angle joint value", 80, 0, 255)
gen.add("joint5_value", int_t, 0, "joint5 angle joint value", 80, 0, 255)

exit(gen.generate(PACKAGE, NODE, "bas_arm_driver_tester"))
